src/igvfd/upgrade/file.py

```diff
@@ -548,3 +548,37 @@ def tabular_file_17_18(value, system):
         notes += f' This file\'s content_type was pipeline inputs, but has been upgraded to pipeline parameters.'
     if notes.strip() != '':
         value['notes'] = notes.strip()
+
+
+@upgrade_step('reference_file', '19', '20')
+def reference_file_19_20(value, system):
+    # https://igvf.atlassian.net/browse/IGVF-2973
+    notes = value.get('notes', '')
+    # If upgrading from external is True
+    if value['external'] is True:
+        value['externally_hosted'] = True
+        # Inherit source_url if available (it is not required metadata)
+        if value.get('source_url'):
+            value['external_host_url'] = value['source_url']
+            notes += f' This file was previously marked as `external` as True with a `source_url` of {value["source_url"]}. It has been upgraded to be `external_host_url` as True with `externally_hosted_url` of {value["source_url"]}.'
```

Comment:
It has been upgraded to be `externally_hosted` as True with `external_host_url` of {value["source_url"]}.